### ISSUE TITLE
No tests for organizations

### ISSUE THREAD

**Issue by Lyla-Fischer** [fablabbcn/fablabs#366]
It looks like there is zero testing for organizations in the existing code base - unless I am missing something.

**Comment by openp2pdesign** [comment]
Yes, `Organizations` are a very recent idea that still has to be structured, it did not make any sense (and there was no time) for adding tests for something that we are still exploring.
